src/main/scala/ai/privado/languageEngine/java/processor/JavaProcessor.scala

```diff
@@ -123,18 +124,45 @@ object JavaProcessor {
     * @return
     */
   def createJavaCpg(processedRules: ConfigAndRules, sourceRepoLocation: String, lang: String): Either[String, Unit] = {
-
     println(s"${Calendar.getInstance().getTime} - Processing source code using ${Languages.JAVASRC} engine")
     if (!config.skipDownloadDependencies)
       println(s"${Calendar.getInstance().getTime} - Downloading dependencies and Parsing source code...")
     else
       println(s"${Calendar.getInstance().getTime} - Parsing source code...")
     cpgconfig = Config(inputPath = sourceRepoLocation, fetchDependencies = !config.skipDownloadDependencies)
     val xtocpg = JavaSrc2Cpg().createCpgWithOverlays(cpgconfig)
+    if (config.showUnresolvedFunctionsReport)
+      reportUnresolvedMethods(xtocpg)
     println(
       s"${TimeMetric.getNewTime()} - Base processing done in \t\t\t\t- ${TimeMetric.setNewTimeToLastAndGetTimeDiff()}"
     )
     processCPG(xtocpg, processedRules, sourceRepoLocation)
   }
 
+  def reportUnresolvedMethods(xtocpg: Try[Cpg]): Unit = {
+    var unresolvedSignatures = 0
+    var unresolvedNamespaces = 0
+    var total = 0
+    xtocpg match {
+      case Success(cpg) => {
+        total = cpg.call.methodFullName.l.length
+        unresolvedSignatures = cpg.call.methodFullName("(?i)(.*)(unresolved)(signature)(.*)").l.length
+        unresolvedNamespaces = cpg.call.methodFullName("(?i)(.*)(unresolved)(namespace)(.*)").l.length
+      }
+    }
+    println()
+    println("---------------------------------------------------------------------------------------------------------")
+    println("Total number of function calls: " + total)
+
+    var percentage = (unresolvedNamespaces * 100) / total
+    println("Calls with unresolved namespace: " + unresolvedNamespaces + " | " + percentage + "%")
+    percentage = (unresolvedSignatures * 100) / total
+    println("Calls with unresolved signatures: " + unresolvedSignatures + " | " + percentage + "%")
+
+    val resolved = (total-(unresolvedSignatures+unresolvedNamespaces))
```

Comment:
@gaurav-gogia As I mentioned unresolvedSignature is a superset of unresolvedNamespace.
Change to formula to `total - unresolvedSignatures`